skip-tests.json

```diff
@@ -1254,6 +1254,7 @@
         }
     },
     "p2b": {
+        "httpsProxy": "http://51.83.140.52:11230",
```

Comment:
whats this for?

Reply:
@samgermain I was trying to run the tests though a proxy to see if it fixed the bad URL error but didn't work